src/graphql/acsys/mod.rs

```diff
@@ -350,7 +343,7 @@ fn strip_source(drf: &str) -> &str {
 // a weak form of extracting; we should really have a DRF parser.
 
 fn device_name(drf: &str) -> &str {
-    drf[0..drf.find(['[', '@', '<']).unwrap_or(drf.len())].trim_end()
+    drf[0..drf.find(['@', '<']).unwrap_or(drf.len())].trim_end()
```

Comment:
This is a big change. Do we care that we've moved from just device name to device+property+range+field?